### ISSUE TITLE
Big Drop in KH/s

### ISSUE THREAD

**Issue by petermash** [vertcoin-project/one-click-miner-vnext#213]
My KH/s has halved, I was getting around 820KH/s then I had to stop mining. I then restarted an item got blocked by F-Secure which stopped it from mining, I then allowed the item in F-Secure and it started mining again but it has halved the KH/s to 405.2KH/s. I have then turned off F-Secure and any other  it still runs at 405.2KH/s. I have restarted the PC and its still the same. I have looked at GPU-Z and both of my GPUs are running at 100% which they were when I was getting 820KH/s but its just been halved. I am using the Zergpool and that also has not paid out when I have 8.5 VTC in the pending pool payout. I am new to this so not sure what is going on really.
I was expecting to have the same hash rate as nothing has changed at my end, I didn’t know why it has dropped. 

I have tried to restart my PC, close any programs that might affect the GPUs performance and nothing changes it. It stills runs at around 400KH/s, so it happens every time I start it. But only started today.

All the time I just start the One click Miner 

I am using vertcoin-ocm-2.0-rc14-windows-x64

On a windows using a AMD RX 590 and a Nvidia 1070

Thanks for any help.

Regards 

Peter

**Comment by vertiond** [comment]
If it's running at half the hash rate, somehow two instances of the miner are running.  Do you have the Auto Start checkbox selected?
Zergpool pays out when the 'Total Unpaid' amount (as seen from https://zergpool.com/?address= reaches the equivalent of 0.0001 BTC or 0.00002 BTC on Sundays.

**Comment by petermash** [comment]
Hi,

Firstly thanks for your help. I did have auto start on but after your email I turned it off I also did some testing by taking out the GPU’s and seeing what they did by themselves and then installing them both again to see if it would start giving me the hash rate for both again but it didn’t. Iv attached some images showing the AMD 590 by its self and then the 1070 and then both of them installed. Which card is being use is in the top left of the image also the image is named by what card is being used.
